### ISSUE TITLE
Error in loop with this plugin

### ISSUE THREAD

**Issue by RodrigoKazuma** [Prolo1/Character-Morpher#3]
I downloaded this plugin for HS2, but an error started to show.

Here is my log to help you:

[output_log.txt](https://github.com/Prolo1/Character-Morpher/files/9425432/output_log.txt)

**Comment by Prolo1** [comment]
I have seen that error before as well and also tried to fix it but I couldn't... the issue is made from a hook called from the Heelz mod. https://github.com/hooh-hooah/IL_Heelz/blob/832cd279a21ea71789a62e3a3a76ace977c1d882/HS2_Heelz/Plugin.cs#L69-L82

In my mod, I make an extra ChaControl object which takes care of loading the morph target. I have tried disabling the object entirely to stop it from calling LateUpdate but Unity seems to not care ¯\_(ツ)_/¯. every time LateUpdate is called on a character I assume fullbodyIK doesn't exist causing that error on line # 75 (literally the only thing w/o a check).

**Comment by Prolo1** [comment]
just remove the heelz plugin (for now) until I find a permanent fix or just ignore the error (the error isn't actually breaking anything).

**Comment by Prolo1** [comment]
Did a bit of digging on this issue and I figured out a way around it. Turns out the issue was caused because the LateUpdateForced function was being called from a completely different object because when I instantiated the ChaControl object, it adds the object to a list of characters internally and updates it that way (in a separate location for no apparent reason). I just had to remove it from that internal list and no more errors.  May have even fixed another issue by accident because of that too.
